src/AppBundle/Entity/Todo.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Todo
+ *
+ * @ORM\Table(name="todo")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\TodoRepository")
+ */
+class Todo
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="body", type="text")
+     */
+    private $body;
+
+    /**
+     * @var bool
+     *
+     * @ORM\Column(name="completed", type="boolean")
+     */
+    private $completed;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Acme\UserBundle\Entity\User", inversedBy="todos")
+     * @ORM\JoinColumn(name="user_id", referencedColumnName="id")
+     */
+    private $user;
+
+    /**
+     * Get id
+     *
+     * @return integer 
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set body
+     *
+     * @param string $body
+     * @return Todo
+     */
+    public function setBody($body)
+    {
+        $this->body = $body;
+
+        return $this;
+    }
+
+    /**
+     * Get body
+     *
+     * @return string 
+     */
+    public function getBody()
+    {
+        return $this->body;
+    }
+
+    /**
+     * Set completed
+     *
+     * @param boolean $completed
+     * @return Todo
+     */
+    public function setCompleted($completed)
+    {
+        $this->completed = $completed;
+
+        return $this;
+    }
+
+    /**
+     * Get completed
+     *
+     * @return boolean 
+     */
+    public function getCompleted()
+    {
+        return $this->completed;
+    }
+
+    /**
+     * Set user
+     *
+     * @param \Acme\UserBundle\Entity\User $user
+     * @return Todo
+     */
+    public function setUser(\Acme\UserBundle\Entity\User $user = null)
```

Comment:
Oh, thank you!
Btw, does this current format lead to possible security leaks? Or is it just a best practice to avoid repeating the name in each action?